### PR TITLE
Wip/mqtt/init - tracking mqtt experiments

### PR DESCRIPTION
**Pull request by bewest**
The corresponding uploader changes (there are a few actually)

The protobuf model:
https://github.com/nightscout/android-uploader/blob/master/core/src/main/java/com/nightscout/core/model/Download.proto

Feel free to close/ignore if this gets in the way of things, I'll move it to my personal fork if it becomes a distraction.

### CODE REVIEW

File: lib/entries.js [hunk 1]
```diff
@@ -167,6 +167,11 @@ function storage(name, storage, pushover) {
     });
   }
 
+  function writeStream (opts) {
```
Comment (jasoncalabrese) [inline lib/entries.js]: doesn't look like this is used by anything

**Comment by bewest** [comment]
The rebasing includes re-plays on top of `wip/bootevent/init`.

File: lib/models/entries.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+function make_field ( ) {
+}
+
+function sanitizer (opts) {
+  
+}
+
+function index (opts) {
+  function calc (data) {
+    var o = lo.pick(data, opts.keys);
```
Comment (jasoncalabrese) [inline lib/models/entries.js]: should `lo` be `_`?

File: lib/mqtt.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                      }));
+                    iter_mqtt_record_stream(packet, 'cal', toCal)
+                      .pipe(core.persist(function empty(err, result) {
+                          console.log("DONE WRITING Cal TO MONGO", err, result.length);
+                      }));
+                    iter_mqtt_record_stream(packet, 'meter', toMeter)
+                      .pipe(core.persist(function empty(err, result) {
+                          console.log("DONE WRITING Meter TO MONGO", err, result.length);
+                      }));
+                    iter_mqtt_record_stream(packet, 'sensor', toSensor)
```
Comment (jasoncalabrese) [inline lib/mqtt.js]: need to get `sensor` entries merged with `sgv`s

File: lib/pebble.js~ [hunk 4]
```diff
@@ -0,0 +1,187 @@
+'use strict';
```
Comment (jasoncalabrese) [inline lib/pebble.js~]: an extra file, note the trailing ~

**Comment by bewest** [comment]
W00t exciting

**Pull request merged**